pkg/recipes/terraform/execute_test.go

```diff
@@ -168,7 +168,10 @@ func TestSetEnvironmentVariables(t *testing.T) {
 
 			e := executor{}
 
-			err = e.setEnvironmentVariables(ctx, tf, tc.opts.EnvConfig)
+			if tc.opts.EnvConfig != nil {
```

Comment:
Why is this check needed in the unit test?

Reply:
removed it.